tests/system_tests/services/tiled_config/config.yaml

```diff
@@ -0,0 +1,19 @@
+authentication:
+  # The default is false. Set to true to enable any HTTP client that can
+  # connect to _read_. An API key is still required to write.
+  allow_anonymous_access: true
+trees:
+  - path: /
+    tree: catalog
+    args:
+      uri: "sqlite:////storage/catalog.db"
+      writable_storage: "sqlite:////storage/catalog.db"
+      # This creates the database if it does not exist. This is convenient, but in
+      # a horizontally-scaled deployment, this can be a race condition and multiple
+      # containers may simultaneously attempt to create the database.
+      # If that is a problem, set this to false, and run:
+      #
+      # tiled catalog init URI
+      #
+      # separately.
```

Comment:
```suggestion
```

This config is only for a non-scaled deployment so don't need to worry about this